CMakeLists.txt

```diff
@@ -85,6 +91,7 @@ endif()
 if(${OSQUERY_BUILD_DISTRO} STREQUAL "rhel6")
   set(GCC_RUNTIME "/opt/rh/devtoolset-3/root/usr/")
   WARNING_LOG("Setting RHEL6 GCC runtime: ${GCC_RUNTIME}")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} --gcc-toolchain=${GCC_RUNTIME}")
```

Comment:
Without the toolchain set in C flags clang and glogs autoconf will fail simple "can I compile" tests.